### DISCUSSION TITLE
Which brach of software engineering to take

### DISCUSSION THREAD

**Discussion by Hartwell9White** [community/community#125141] [New to GitHub]
Hello guys im new to software engineering. i have a quite artistic mind and i wonder which path of the art i should take

**Comment by PerezChris99** [comment]
Hello there White, since you are artistic, you should take a look into front end development (its a branch of software engineering that focuses on building user interfaces and making them responsive)

**Comment by PerezChris99** [comment]
To do this, take on front end technologies like html+css, javascript, then move on to a front end framework like react, vue, angular.
and also look at tailwind CSS..have fun

**Comment by PerezChris99** [comment]
ooh almost forgot, you should look into version control. (how to use git and github) remember , don't skip the boring parts
